pkg/lib/analytic/time_test.go

```diff
@@ -0,0 +1,64 @@
+package analytic_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/authgear/authgear-server/pkg/lib/analytic"
+	"github.com/authgear/authgear-server/pkg/util/periodical"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestGetDateListByRange(t *testing.T) {
```

Comment:
It seems that this function treat rangeTo as inclusive. Would exclusive rangeTo be more conventional? Or we should name the function `GetDateListByRangeInclusive`.